pagination/page_request.go

```diff
@@ -0,0 +1,29 @@
+package pagination
+
+const defaultPageSize = 20
+
+type PageRequest struct {
+	Page     int // page requested (default 1)
+	PageSize int // Number of items per page
+}
+
+func NewPageRequest(page, pageSize int) PageRequest {
```

Comment:
suggestion: by reading the code in the SWAN PR, I feel like a better name would be `Request` so that it's called `pagination.Request` from outside. What do you think?

```suggestion
type Request struct {
	Page     int // page requested (default 1)
	PageSize int // Number of items per page
}

func NewRequest(page, pageSize int) Request {
```

Reply:
Praise: nice suggestion